src/pulselistener/pulselistener/monitoring.py

```diff
@@ -222,3 +222,7 @@ def send_report(self):
 
         # Reset stats
         self.stats = {}
+
+
+# Shared monitoring manager
+task = Monitoring(7 * 3600)
```

Comment:
I'd keep calling this `task_monitoring` so you don't have to do `import as`. I also like this name more than `task`